packages/py-api/py_api/controllers/hackathon_teams_controller.py

```diff
@@ -0,0 +1,124 @@
+import json
+import random
+import string
+from typing import Literal
+
+from bson.json_util import dumps
+from bson.objectid import ObjectId
+from fastapi.responses import JSONResponse
+from py_api.database.initialize import t_col
+from py_api.models import UpdateTeam
+from py_api.utilities.parsers import filter_none_values
+
+
+class TeamsController:
+    @classmethod
+    def create_team(cls, team_name: str, user_id: str) -> Literal[True] | Literal[False]:
```

Comment:
This is a controller, it should return a `JSONResponse`. Extract whatever functionality you need in a function/method.

Reply:
The initial idea was not to have an endpoint for creating new teams, this is why this function is not returning a JSONResponse. We check the return value and return the  JSONResponse, where this function is called, for example in the `ParticiapntsController.add_participant()` function.

Reply:
Okay. That's alright. But instead of returning a boolean, return the expected object or `None`